src/realm/object-store/sync/impl/sync_metadata.cpp

```diff
@@ -51,6 +52,7 @@ static const char* const c_sync_device_id = "device_id";
 
 /* User Profile keys */
 static const char* const c_sync_profile = "profile";
+static const char* const c_sync_profile_data = "profile_data";
 static const char* const c_sync_profile_name = "name";
 static const char* const c_sync_profile_first_name = "first_name";
```

Comment:
Are these used now? Seems like we're hardcoding the values in sync_user.hpp?

Reply:
Suppose we could have globals for these keys, though it's not exactly a hot path.